tests/functional/tests/bundles/base.yaml

```diff
@@ -3,7 +3,6 @@ applications:
   rabbitmq-server:
     charm: rabbitmq-server
     num_units: 1
-    constraints: root-disk=8G cores=2 virt-type=virtual-machine
```

Comment:
We used virtual machine here for some reason. It's safe to drop it? Cannot we use both arm and vm?

Reply:
No, we can't use both arm and vm unfortunately:

> The tests are simplified because the arm64 runners don't have kvm support (so we can't deploy the original bundle with virtual-machine type machines).

Reply:
It's safe to drop it; it just means that we can't test as many cases in the func tests.